ivy/functional/frontends/torch/tensor.py

```diff
@@ -1846,6 +1846,12 @@ def index_fill(self, dim, index, value):
         arr = torch_frontend.moveaxis(self, 0, dim)
         return arr
 
+    @with_unsupported_dtypes(
+        {"2.0.1 and below": ("uint8", "uint16", "bfloat16", "float16")}, "torch"
+    )
```

Comment:
How did you arrive at this list of unsupported types? The function in `miscellaneous_ops` seems to support these, so the instance method ideally should too

Reply:
I actually got most of this from the backend implementation in ivy/functional/backends/torch/experimental/statistical.py. The other came up as a solution to a error I was facing while testing.

Reply:
ok, thank you. I will do that rn.